apps/scan/backend/src/app.ts

```diff
@@ -86,13 +86,12 @@ function buildApi(
     async configureFromBallotPackageOnUsbDrive(): Promise<
       Result<void, BallotPackageConfigurationError>
     > {
-      assert(!store.getElectionDefinition(), 'Already configured');
-      const [usbDrive] = await usb.getUsbDrives();
-      assert(usbDrive?.mountPoint !== undefined, 'No USB drive mounted');
-
       const authStatus = await auth.getAuthStatus(
         constructAuthMachineState(workspace)
       );
+      assert(!store.getElectionDefinition(), 'Already configured');
+      const [usbDrive] = await usb.getUsbDrives();
+      assert(usbDrive?.mountPoint !== undefined, 'No USB drive mounted');
```

Comment:
Can you remind me why this has to move under the auth check?

Reply:
Ah yeah this move isn't strictly necessary!

I've just been getting in the habit of making auth status retrievals the first step in backend handlers since I eventually want to move auth status retrievals into auth check middlewares/decorators. So having auth status retrievals come first is more representative of the planned long-term state.